### PR TITLE
Move cluster-essentials files to a new directory dedicated for installation resources

### PR DESCRIPTION
**Pull request by mjakobczyk**
**Description**

Changes proposed in this pull request:

- move `cluster-essetials`'s `/files` directory to `/installation/resources` as it's going to be a central point from which resources will be taken for a new installation
- make new directory clutter-free and divide by first: resource type and second: component

**Related issue(s)**

https://github.com/kyma-incubator/hydroform/issues/177

### CODE REVIEW

**Comment by Halamix2** [comment]
Recent changes in projwjob definitions trigerred some unnecessary jobs in “Expected” state, `/retest` fixes that

**Review by k15r — APPROVED** [review]

**Review by strekm — APPROVED** [review]
/meow

**Review by suleymanakbas91 — APPROVED** [review]

File: installation/resources/crds/service-catalog/servicebindingusages.servicecatalog.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (jasiu001) [inline installation/resources/crds/service-catalog/servicebindingusages.servicecatalog.crd.yaml:1]: if we follow by resources rules then `servicebindingusages` should be in `service-catalog-addons`
see [resourses](https://github.com/kyma-project/kyma/tree/main/resources/service-catalog-addons/charts).
if there are no rules or rules ar different please let me know

File: installation/resources/crds/service-catalog/usagekinds.servicecatalog.crd.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (jasiu001) [inline installation/resources/crds/service-catalog/usagekinds.servicecatalog.crd.yaml:1]: this should be part of `service-catalog-addons` (see first comment)

File: resources/service-catalog/templates/backendmodule.yaml [hunk 3]
```diff
@@ -1,4 +1,6 @@
+{{- if .Capabilities.APIVersions.Has "ui.kyma-project.io/v1alpha1" }}
```
Comment (jasiu001) [inline resources/service-catalog/templates/backendmodule.yaml:1]: imo it also should be part of `service-catalog-addons`

**Review by jasiu001 — APPROVED** [review]

**Review by tgorgol — APPROVED** [review]

**Review by Maladie — APPROVED** [review]

**Review by dariadomagala — APPROVED** [review]

**Review by berkanercan — APPROVED** [review]
Looking ok

**Pull request merged**